completion/GS1.cwl

```diff
@@ -18,4 +18,11 @@ code=#EAN-8,EAN-13
 scale=%<factor%>
 scale_to_font#true,false
 add_control#true,false
-#endkeyvals
\ No newline at end of file
+#endkeyvals
+
+# expl3 interface
+\GS_cut_EAN_control_digit:N %<⟨seq var⟩%>#/%expl3
+\GS_set_code_digit_seq:Nn %<⟨seq var⟩%> {%<⟨tokens⟩%>}#/%expl3
+\GS_set_EAN_control_digit:N %<⟨seq var⟩%>#/%expl3
+\GS_use_as_EAN_barcode:N %<⟨seq var⟩%>#/%expl3
+\int_set_to_EAN_control_digit:NN %<⟨integer⟩ ⟨seq var⟩%>#/%expl3
```

Comment:
Maybe
```
\int_set_to_EAN_control_digit:NN %<⟨integer⟩%> %<⟨seq var⟩%>#/%expl3
```
is better, but I didn't check if it's supported or not.

Reply:
If the user does not use placeholders, that syntax leaves an unnecessary extra space after the command